### DISCUSSION TITLE
any way to download some parts of Sentinel-1 SLC ?

### DISCUSSION THREAD

**Discussion by soroushyasini** [asfadmin/Discovery-asf_search#271] [Q&A]
hi to everyone, my study area is located in second sub-swath so I really don't need other sub-swaths to download and additionally I just need VV polarization to download. as my network is not really suitable for large scale downloading, i have to reduce amount of downloading data. is there any way to consider ?

**Comment by jhkennedy** [comment]
@soroushyasini I'd suggest checking out our [Sentinel-1 burst collection](https://asf.alaska.edu/datasets/data-sets/derived-data-sets/sentinel-1-bursts/) which will allow you download individual busts. Unfortunately not all data is available in the catalog (we're working on back population of the catalog for the entire Sentiel-1 archive) but you can use the extractor directly yourself to get any burst in the archive:
https://sentinel1-burst-documentation.asf.alaska.edu/
